modules/monitoring-about-specifying-limits-and-requests-for-monitoring-components.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * monitoring/configuring-the-monitoring-stack.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="about-specifying-limits-and-requests-for-monitoring-components_{context}"]
+= About specifying limits and requests for monitoring components
+
+You can configure resource limits and request settings for core platform monitoring components and for the components that monitor user-defined projects, including the following:
```

Comment:
```suggestion
You can configure resource limits and request settings for core platform monitoring components and for the components that monitor user-defined projects, including the following components:
```
From IBM Style Word Usage: "To introduce a list, procedure, or subsequent items, always use _following_ as an adjective before a noun. For example, do not write “Complete the following” or ”The functions include the following”. Instead, write “Complete the following steps” or “The adapter has the following functions”."

Reply:
Good catch. Thanks!